strongbox-web-core/src/main/java/org/carlspring/strongbox/config/SecurityConfig.java

```diff
@@ -53,6 +55,8 @@ protected void configure(HttpSecurity http)
         http.sessionManagement()
             .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
             .and()
+            .exceptionHandling().authenticationEntryPoint(basicAuthenticationEntryPoint())
```

Comment:
But I am not sure that `user-agent` is the best way, and maybe you will have some thoughts about this.